lang/ja.yml

```
@@ -83,110 +83,110 @@ INSPECTOR_TOGGLED: "検査官モードが{有効|無効}になりました。"
 INTEGRATION_ERROR: "できません {初期化|無効化} {0} ロギング。"
 INTEGRATION_SUCCESS: "{0} 正常にロギング {初期化|無効}。"
 INTEGRATION_VERSION: "無効 {0} バージョンが見つかりました。"
-INTERACTIONS_HEADER: "プレイヤーの相互作用"
+INTERACTIONS_HEADER: "ユーザーの相互作用"
 INVALID_ACTION: "それは有効なアクションではありません。"
 INVALID_BRANCH_1: "プラグインのバージョンが無効です（ブランチが設定されていません）。"
-INVALID_BRANCH_2: "続行するには、プロジェクトブランチを「開発」に設定します。"
+INVALID_BRANCH_2: "続行するには、プロジェクトブランチを「開発」に設定してください。"
 INVALID_BRANCH_3: "開発コードを実行すると、データが破損する可能性があります。"
 INVALID_CONTAINER: "最初に有効な貨物ブロックを調べてください。"
 INVALID_DONATION_KEY: "寄付キーが無効です。"
 INVALID_INCLUDE: "「{0}」は無効な値です。"
 INVALID_INCLUDE_COMBO: "これは無効な型の組み合わせです。"
 INVALID_RADIUS: "有効な半径を入力してください。"
 INVALID_SELECTION: "{0} 選択が見つかりません。"
-INVALID_USERNAME: "「{0}」は無効なユーザー名です。"
+INVALID_USERNAME: "\"{0}\"は無効なユーザー名です。"
 INVALID_WORLD: "有効なワールドを指定してください。"
 LATEST_VERSION: "最新バージョン: {0}"
 LINK_DISCORD: "Discord: {0}"
 LINK_DOWNLOAD: "ダウンロード: {0}"
 LINK_PATREON: "Patreon: {0}"
 LINK_WIKI_BLOCK: "ブロック名: {0}"
 LINK_WIKI_ENTITY: "エンティティ名: {0}"
-LOGGING_ITEMS: "{0} ログに残されたアイテム。お待ちください..."
+LOGGING_ITEMS: "{0}件のログにアイテムが残されています。しばらくお待ちください..."
 LOGGING_TIME_LIMIT: "記録制限時間に達しました。データを破棄してシャットダウンします。"
-LOOKUP_BLOCK: "{0} {配置|壊れた} {1}。"
-LOOKUP_CONTAINER: "{0} {追加|削除} {1} {2}。"
-LOOKUP_HEADER: "{0} 結果"
-LOOKUP_INTERACTION: "{0} {クリック|殺された} {1}。"
-LOOKUP_ITEM: "{0} {拾った|落とした} {1} {2}。"
-LOOKUP_LOGIN: "{0} ログに記録されます {in|out}。"
-LOOKUP_PAGE: "ページ {0}"
-LOOKUP_PROJECTILE: "{0} {投げた|ショット} {1} {2}。"
-LOOKUP_ROWS_FOUND: "{0} {行|行} 見つかりました。"
-LOOKUP_SEARCHING: "ルックアップ検索。お待ちください..."
-LOOKUP_STORAGE: "{0} {預け入れ|撤回} {1} {2}。"
-LOOKUP_TIME: "{0} 前に"
-LOOKUP_USERNAME: "{1}が{0}としてログイン"
-MAXIMUM_RADIUS: "最大 {ルックアップ|ロールバック|復元} 半径は {0}。"
+LOOKUP_BLOCK: "{0}が{1}を{配置|破壊}した。"
+LOOKUP_CONTAINER: "{0}が{2} {1}を{追加|取り出}した。"
+LOOKUP_HEADER: "{0} の結果"
+LOOKUP_INTERACTION: "{0}が{1}を{クリック|破壊}した。"
+LOOKUP_ITEM: "{0}が{2} {1}を{拾った|落とした}。"
+LOOKUP_LOGIN: "{0}の{入力|出力}がログに記録されます。"
+LOOKUP_PAGE: "{0} ページ目"
+LOOKUP_PROJECTILE: "{0}が{2} {1}を{投げた|撃った}。"
+LOOKUP_ROWS_FOUND: "{0}行見つかりました。"
+LOOKUP_SEARCHING: "検索中。しばらくお待ちください..."
+LOOKUP_STORAGE: "{0}が{2} {1}を{預け入れ|引き出し}た。"
+LOOKUP_TIME: "{0}前"
+LOOKUP_USERNAME: "{0}が{1}としてログイン"
+MAXIMUM_RADIUS: "最大半径の{0}中で{検索|ロールバック|復元}をします。"
 MISSING_ACTION_USER: "そのアクションを使用するには、ユーザーを指定してください。"
-MISSING_LOOKUP_TIME: "時間を指定してください({ルックアップ|ロールバック|復元})"
+MISSING_LOOKUP_TIME: "時間を指定してください({検索|ロールバック|復元})"
 MISSING_LOOKUP_USER: "ユーザーを指定するか、 {ブロック|半径}を指定してください。"
 MISSING_PARAMETERS: "\"{0}\"のパラメーターは存在していません。"
-MISSING_ROLLBACK_RADIUS: "指定しませんでした {ロールバック|復元} 半径。"
-MISSING_ROLLBACK_USER: "指定しませんでした {ロールバック|復元} ユーザー。"
-MYSQL_UNAVAILABLE: "接続できません MySQL サーバ。"
+MISSING_ROLLBACK_RADIUS: "{ロールバック|復元}対象の半径を指定しませんでした。"
+MISSING_ROLLBACK_USER: "{ロールバック|復元}対象のユーザーを指定しませんでした。"
+MYSQL_UNAVAILABLE: "MySQLサーバーに接続できません"
 NO_DATA: "{0}でデータが見つかりません。"
-NO_DATA_LOCATION: "番号 {データ|トランザクション|相互作用|メッセージ} 見つかった。"
+NO_DATA_LOCATION: "ここで{データ|入れ替え|相互作用|メッセージ}は見つかりませんでした。"
```

```suggestion
NO_DATA_LOCATION: "{データ|トランザクション|相互作用|メッセージ}は見つかりませんでした。"
```

データに関する語句で、トランザクションは入れ替えなどのニュアンスでは使われていません。
ACID特性などの処理方式の方の単語で使われているので
トランザクションはそのままが良いと思う

The term is used in the sense of data, and transactions are not used in the sense of replacement or other nuances.
The term is used to describe ACID characteristics and other processing methods.
Transaction is better used as it is.

参考: https://swri.jp/glossary/%E3%83%88%E3%83%A9%E3%83%B3%E3%82%B6%E3%82%AF%E3%82%B7%E3%83%A7%E3%83%B3

---

https://github.com/PlayPro/CoreProtect/blob/master/lang/en.yml#L133

を見ると否定形としての「No」が使われているので
最後が「見つかりません」なら先頭に文字は必要ありません。

shows that “No” is used as a negative form, so the first letter is not necessary if the last is “Not found.
If the last word is “not found”, the first letter is not necessary.

機械翻訳では、「No」を「No.（番号）」と誤訳することがよくある。
In machine translation, there is often a mistranslation of “No” into “No. (number)”.